### PR TITLE
docs: list `ibc-rs` adoption on README

### PR DESCRIPTION
**Pull request by rnbguy**
Closes: #XXX

## Description

Rendered: https://github.com/cosmos/ibc-rs/blob/rano/ibc-rs-adaption-list/README.md#ibc-rs-in-the-wild

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests.
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,30 @@ development.
...
+[namada]: https://namada.net
+[basecoin-rs]: https://github.com/informalsystems/basecoin-rs
+[hermes]: https://hermes.informal.systems
+[sov-ibc]: https://github.com/informalsystems/sovereign-ibc
+[substrate-ibc]: https://github.com/octopus-network/substrate-ibc
+[zk-ibc]: https://www.zkibc.com
+
 ## License
 
 Copyright © 2022 Informal Systems Inc. and ibc-rs authors.
```
Comment (seanchen1991) [inline README.md]: I know this isn't part of the PR, but I noticed it anyway: should we update the copyright year here?

File: README.md [hunk 2]
```diff
@@ -73,9 +73,42 @@ development.
...
 - [Protobuf definitions in Rust][ibc-proto-rs]
 
+## `ibc-rs` in the wild
+
+The IBC protocol is integrated in numerous networks and projects. The `ibc-rs` implementation specifically
+is adopted for IBC integration, testing, or development in the following use-cases.
+
+|  # | Integration use-Case   |     Team      | Used-Modules |                 Links                 |          Phase           |
+|---:|------------------------|:-------------:|--------------|:-------------------------------------:|:------------------------:|
+|  1 | Solana                 | ComposableFin | all          |    [Picasso Network][pica-network]    |    :white_check_mark:    |
```
Comment (JafarAz) [inline README.md]: Could you please rename the Team from ComposableFin to either: `Composable` or `Composable Foundation`

Comment (adizere) [reply README.md]: done

**Review by JafarAz — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -73,9 +73,42 @@ development.
...
 - [Protobuf definitions in Rust][ibc-proto-rs]
 
+## `ibc-rs` in the wild
+
+The IBC protocol is integrated in numerous networks and projects. The `ibc-rs` implementation specifically
+is adopted for IBC integration, testing, or development in the following use-cases.
+
+|  # | Integration use-case |     Team      | Used-Modules |                 Links                 |          Phase           |
+|---:|----------------------|:-------------:|--------------|:-------------------------------------:|:------------------------:|
+|  1 | Solana               |  Composable   | all          |    [Picasso Network][pica-network]    |    :white_check_mark:    |
```
Comment (JafarAz) [inline README.md]: If you want to be specific with the use cases:

- Solana IBC implementation
- Ethereum IBC relayer
- Polkadot SDK IBC implementation

Comment (merkletreeibc) [reply README.md]: Suggested language change for the introduction:

The [IBC Protocol](https://ibcprotocol.dev/) is integrated in numerous networks and projects. The `ibc-rs` implementation, in particular, is used for IBC integration, testing, or development for some of the most technically compelling use cases.

Comment (rnbguy) [reply README.md]: @JafarAz can you tell us which ibc-rs components you are using in each of these integrations?

File: README.md (hunk 1 above)
Comment (rnbguy) [reply README.md]: 6d9f785

File: README.md (hunk 3 above)
Comment (vmarkushin) [reply README.md]: @rnbguy we're using the core library, 07-tendermint and 08-wasm clients, and transfer module

**Review by seanchen1991 — APPROVED** [review]
Looks good to me 👍

**Review by adizere — APPROVED** [review]

**Pull request merged**